### ISSUE TITLE
As Vaults user creating a vault, I need to be able to specify how much collateral to lock up and see corresponding IST to receive

### ISSUE THREAD

**Issue by otoole-brendan** [Agoric/dapp-inter#3]
## What is the Problem Being Solved?

Implement a 'Configure Vault' dialog box within 'Create a Vault' view where the user can specify how much collateral they wish to lock up in order to receive corresponding amount of IST.

## Description of the Design
-Step 1: Pick your vault is covered by [#2](https://github.com/Agoric/dapp-inter/issues/2)

- Step 2: Configure Vault (See Figma design for reference: https://www.figma.com/file/r0WcCjyS0LAOalFGV4DSuk/agoric-v1?node-id=1876%3A12067&t=M1K9MKcyPF9GJnCO-0)

action/intended behavior -> user can edit 2 fields:
1) 'Collateral/Atom to Lock Up'
2) 'IST to Receive'

- If the user edits field 1, field 2 will auto-calculate and vice versa.

- The 3rd field, 'Collateralization Percent' is not editable but will dynamically update in line with what user inputs into other two fields.

- Validation on 'Collateral To Lock Up' field - user must have sufficient ATOM in their wallet on Agoric Chain

The user needs to be prompted to 'Connect Wallet' before we can know if they have a balance on Agoric Chain (or if they have an Agoric Smart Wallet at all).

The user will also be able to click a persistent 'Transfer Funds' button which will take them to Agoric Smart Wallet where they can carry out that function before returning to Vaults UI.

We need to be able to guide the users with prompts so they know at all times what they should be doing/where they should be going/checking, action they're supposed to be carrying out. These pop-up dialogs/prompts are visible in Figma design but they are currently a nice to have and not critical to MVP if time is tight: 
prompt user to connect wallet to continue then serve relevant informational pop-ups/CTA as and when they need to: 
a) connect Keplr Wallet 
b) create a new wallet 
c) send dapp connection request and approve in wallet 
d) approve offer in wallet
e) confirmation of vault created when they return to UI

## Security Considerations
Unknown
## Test Plan
Smoke Test after implementation

**Comment by otoole-brendan** [comment]
@samsiegart I have added more detail to include the handoff messaging as it pertains to this function, we could further separate them out though so they're a separate ticket? Thoughts?
